src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -1005,6 +1011,21 @@ def llvm_test_one_input_override(frame, frame_struct):
   return frame
 
 
+def _reduce_string(text, length):
```

Comment:
Maybe we could just do [this](https://github.com/google/clusterfuzz/blob/c2962b66096574bb514cb4646906d45904538b5e/src/python/crash_analysis/stack_parsing/stack_analyzer.py#L593) instead of this function for simplicity.

Reply:
yeah, this is an overkill, plus we won't put long custom string into the crash type, removing this